src/dso_api/dynamic_api/filters/openapi.py

```diff
@@ -0,0 +1,177 @@
+"""OpenAPI logic for dynamic filters.
+
+This generates the most common parameters that are allowed for the filters.
+More fields as possible, since relations can be followed.
+Those fields are not included here.
+"""
+from __future__ import annotations
+
+import re
+
+from schematools.types import DatasetFieldSchema, DatasetTableSchema
+
+from dso_api.dynamic_api.filters import parser
+from dso_api.dynamic_api.filters.parser import QueryFilterEngine
+
+RE_GEOJSON_TYPE = re.compile(r"^https://geojson\.org/schema/(?P<geotype>[a-zA-Z]+)\.json$")
+
+OPENAPI_LOOKUP_TYPES = {
+    "isempty": "boolean",
+    "isnull": "boolean",
+    "like": "string",
+}
+
+OPENAPI_COMMA_LOOKUPS = {"in"}
+
+OPENAPI_SCALAR_EXAMPLES = {
+    "boolean": "use true or false",
+    "string": "text",
+    "integer": "number",
+    "number": "number",
+    "array": "id",
+    "object": {},
+}
+
+OPENAPI_FORMAT_EXAMPLES = {
+    "string": {
+        "date": "use yyyy-mm-dd",
+        "time": "use hh:mm[:ss[.ms]]",
+        "date-time": "use yyyy-mm-dd or yyyy-mm-ddThh:mm[:ss[.ms]]",
+        "uri": "URL",
+    }
+}
+
+OPENAPI_LOOKUP_PREFIX = {
+    "lt": "Less than; ",
+    "lte": "Less than or equal to; ",
+    "gt": "Greater than; ",
+    "gte": "Greater than or equal to; ",
+    "not": "Exclude matches; ",
+}
+
+OPENAPI_LOOKUP_EXAMPLES = {
+    "isempty": "Whether the field is empty or not.",
+    "isnull": "Whether the field has a NULL value or not.",
+    "like": "Filter using wildcards (? and *).",
+    "in": "Filter using comma separated values: val1,val2,valN.",
```

Comment:
These describe syntax, but not semantics.

```suggestion
    "like": "String match using wildcards (? and *).",
    "in": "Value must be one of a comma-separated list: val1,val2,valN.",
```

Reply:
ok, good point. I've updated this to:

```
    "like": "Matches text using wildcards (? and *).",
    "in": "Matches any value from a comma-separated list: val1,val2,valN.",
```

Reply:
Even better!